inspire_matcher/validators.py

```diff
@@ -73,3 +73,27 @@ def authors_titles_validator(record, result):
             title_max_score = current_title_jaccard
 
     return (author_score + title_max_score) / 2 > 0.5
+
+
+def cds_identifier_validator(record, result):
+    """Compute a validation score for the possible match.
```

Comment:
```
"""Ensure that the two records have the same CDS identifier.

This is needed because the search is done only for
``external_system_identifiers.value``, which might cause false positives in
case the matched record has an identifier with the same ``value`` but
``schema`` different from CDS.
```